content/blog/any-terraform-provider/index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Introducing: Support For Using Any Terraform Provider with Pulumi"
+date: 2024-08-29T10:00:00-06:00
+allow_long_title: true
+meta_desc: "Pulumi now supports using any Terraform or OpenTofu provider with Pulumi, in just a few seconds"
+meta_image: meta.png # TODO
+authors:
+  - luke-hoban
+  - ian-wahbe
+  - matt-jeffryes
+tags:
+  - terraform
+  - features
+
+social:
+  twitter: "You can now use ANY Terraform Provider from Pulumi! The entire ecosystem of Terraform/OpenTofu providers is now available to Pulumi users. And all the benefits of Pulumi are available to partners and developers building their own Terraform Providers, with no extra work!"
+  linkedin: |
+    You can now use ANY Terraform Provider from Pulumi! 
+    
+    The entire ecosystem of Terraform/OpenTofu providers is now available to Pulumi users. And all the benefits of Pulumi are available to partners and developers building their own Terraform Providers, with no extra work!
+
+    Check it out at https://www.pulumi.com/registry/packages/terraform-provider/ today.
+
+---
+
+One of our core goals at Pulumi is to provide access to manage any cloud infrastructure with a single unified programming model.  Whether it’s multi-cloud (AWS+Azure+Kubernetes), hybrid cloud (GCP+VMWare+Cisco), or managed services (Databricks+GitHub+Cloudflare), Pulumi makes it easy to deploy and manage infrastructure across all of your cloud environments using any of the 150+ cloud providers in the Pulumi Registry.
+
+We’re excited to take this even further by introducing support for using [_any_ Terraform or OpenTofu provider](https://www.pulumi.com/registry/packages/terraform-provider/) from within your Pulumi programs.  If there is a long tail Cloud or SaaS platform that has a provider for those ecosystems, it now works with Pulumi as well.  And if your organization has built your own custom Terraform or OpenTofu provider to support an internal cloud platform, you can use it from Pulumi as well, without having to publish it to any registry.
+
+<!--more-->
+
+Using any Terraform or OpenTofu provider takes just a few seconds using a single command:
```

Comment:
I might add a section with more backstory/context here. For example:

* Why Terraform providers are the closest to an open cloud provider model (foreshadow)
* How we already leverage Terraform providers - and how this news is actually news, since many folks might have already assumed this was true
* The fact that providers are really just plugins to us, thanks to an extensible provider architecture; in other words, "Pulumi is not just a wrapper on top of Terraform" - this is a natural consequence and advantage of our extensibility model

Sort of feels like the post dives super deep very quickly and, if you're a Pulumian it quickly makes sense, but for others who may not be as familiar with where we've come from and where we're going, may need to tee it up more.

Reply:
Addressed in https://github.com/pulumi/docs/pull/12614/commits/e4a7b9d84d9a3708bbe575b0965e021c75d0c745.